docs/INTEGRATION.md

```diff
@@ -88,6 +88,25 @@ Add the following job to your project's `.drone.yml` pipeline (drone version 1.0
       - hadolint  Dockerfile
 ```
 
+## CircleCI
+
+For CircleCI integration use the [docker orb](https://circleci.com/orbs/registry/orb/circleci/docker).
+
+Update your project's `.circleci/config.yml` pipeline (workflows version 2.1), adding the docker orb and you can call the job docker/hadolint:
+
+```yaml
+orbs:
+  docker: circleci/docker@0.5.13
```

Comment:
Is the version number optional?

Reply:
No, for latest you can specify volatile. Also you can specify only the major number https://circleci.com/docs/2.0/using-orbs/#semantic-versioning-in-orbs

Reply:
maybe this should be 0 or 0.5 or maybe even volatile if it does not break in the future.

Reply:
I've changed it to x.y.z, the way circleci write the orbs examples. it's ok or do you prefer versioning it ?